app/views/games/show.html.erb

```diff
@@ -3,26 +3,40 @@
     $('.chessboard .row img').draggable({
       containment: '.chessboard',
       cursor: 'move',
-      revert: 'invalid'
+      snap: '.row',
+      snapMode: 'center',
+      revert: true
     });
     $('.chessboard .row div').droppable({
-      drop: function( event, ui ) {
-        var dropped = ui.draggable;
+      activeClass: "ui-state-hover",
+      hoverClass: "ui-state-active",
+      drop: function(event, ui) {
+        var chessPiece = ui.draggable;
         var droppedOn = $(this);
```

Comment:
sorry i didn't catch this before,

it's a nice convention to name jQuery objects like: `$droppedOn` so that later in the code someone doesn't do `$(droppedOn)`. by just seeing the `$` in front of the variable name we can safely assume it's a jQuery object and start calling jQuery functions on it, like `empty()`.